templates/footer.php

```diff
@@ -1,11 +1,9 @@
-<nav>
-  <h2 class="visuallyhidden">Useful links</h2>
```

Comment:
I'm wondering if we should keep this hidden `<h2>`? I think this was previously added after accessibility testing, to help screenreaders differentiate the footer links from the other content in the page. Otherwise, they will seem to belong to whatever the previous heading in the page was, which could be e.g. the blog post title.